### ISSUE TITLE
How did you choose features for NN?

### ISSUE THREAD

**Issue by tcsere07** [CuteChibiko/TalkingData#1]
Hello CuteChibiko,
It seems that you did not use all the features you constructed for NN, so I'm curious about how you choose features for NN.

thx.

**Comment by CuteChibiko** [comment]
Hi extremin,

I made about 600 features, then used step-wise method(brute-force attack) to choose features not only for NN but also LGBM model.

**Comment by tcsere07** [comment]
Just like forward feature selection？600features, this is a lot of work！

**Comment by CuteChibiko** [comment]
Yes, mainly forward(adding) feature selection, sometimes deleted one by one.

Most of 600 features are variation, like
'ip',
'app',
'device',
'os',
'channel',
'ip_app',
'ip_device',
'ip_os',
'ip_channel',
'app_device',
・・・

So it's a lot of work for my PC, not for me. :)

**Comment by tcsere07** [comment]
Oh I mean that selecting dozens of features from hundreds of features is a lot of work.

**Comment by CuteChibiko** [comment]
You are right, this feature selection was a lot of work and very time consuming.
I used more than one month X 24 hours.

**Comment by tcsere07** [comment]
Congrats for your LB result!

**Comment by huazai-1994** [comment]
how much RAM memory needed to run the model2? 64GRAM eonough?

**Comment by CuteChibiko** [comment]
Hi huazai-1994,

I did not checked how much memory is used for model2.
But I think you can run it with 64GB Ram.
